src/blocks/Listing/components/ListingVideo.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import Ublock from '../../../components/Ublock/Ublock'
+import Heading from '../../../components/Heading/Heading'
+
+export interface ListingVideoProps {
+    header: string
+    noborder?: boolean
+    data: ListProps[]
+}
+
+export interface ListProps {
+    src: string
+    title: string
+    image?: ImageProps
+}
+
+export interface ImageProps {
+    src: string
+    alt: string
+}
+
+
+const ListingVideo: React.FC<ListingVideoProps> = ({
+    header,
+    noborder,
+    data,
+}): JSX.Element => {
+    return (
+            <Ublock>
+                <header>
+                    <Heading header={header} noborder={noborder} />
+                </header>
+                <div className="b-listing b-listing--video">
+                    <ul itemScope itemType="http://schema.org/ItemList">
+                        {data.map((list, index) => (
```

Comment:
Please use semantic variable names. In this case we could use "listItem" or just "item". "List" is more related to a group of items :)